From: Andreas Tille <tille@debian.org>
Date: Tue, 13 Dec 2022 20:14:41 +0100
Subject: Ignore failing tests for the moment

There are some failing tests where pip is called.
We can't call pip since online access is not permitted.
However, for the moment the exclusions are tagged FIXME
to attract further investigation.

Last-Update: Tue, 5 Dec 2023 11:51:00 +0000
---
 tests/test_examples.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/test_examples.py b/tests/test_examples.py
index 98b2788..8ecef69 100644
--- a/tests/test_examples.py
+++ b/tests/test_examples.py
@@ -16,6 +16,7 @@ from mesonpy._util import chdir
 examples_dir = pathlib.Path(__file__).parent.parent / 'docs' / 'examples'
 
 
+@pytest.mark.xfail(reason="FIXME: Needs more investigation to pass in Debian")
 def test_spam(venv, tmp_path):
     """Test that the wheel for the example builds, installs, and imports."""
     with chdir(examples_dir / 'spam'):
